### ISSUE TITLE
Add support for the new telegram web

### ISSUE THREAD

**Issue by Omeramsc** [meetfranz/recipe-telegram#20]
Add support for the two new official Telegram web apps:
https://webk.telegram.org/
https://webz.telegram.org/

**Comment by rphly** [comment]
You can find it here https://github.com/rphly/Franz-Telegram2-Recipe

**Comment by yuzhakovvv** [comment]
@rphly works for me. Great work!

I guess it should be moved to `meetfranz`. Also maybe you should name it _Telegram Web K_ not _Telegram 2_ (as there is also _Telegram Web Z_)?

**Comment by rphly** [comment]
@yuzhakovvv makes sense, renamed: https://github.com/rphly/franz-telegramwebk-recipe
